src/igvfd/schemas/biosample.json

```diff
@@ -141,79 +112,24 @@
-                    },
-                    "then": {
-                        "properties": {
-                            "life_stage": {
-                                "enum": [
-                                    "adult",
-                                    "child",
-                                    "postnatal",
-                                    "embryonic",
-                                    "unknown"
-                                ]
-                            }
-                        }
-                    }
-                },
-                {
-                    "if": {
-                        "properties": {
-                            "taxa": {
-                                "enum": [
-                                    "Saccharomyces"
-                                ]
-                            }
-                        },
-                        "required": [
-                            "taxa"
-                        ]
-                    },
-                    "then": {
-                        "properties": {
-                            "life_stage": {
-                                "enum": [
-                                    "lag",
-                                    "fermentative",
-                                    "respirative",
-                                    "stationary",
-                                    "decline",
-                                    "unknown"
-                                ]
-                            }
-                        }
-                    }
-                }
-            ]
         }
     },
     "properties": {
         "schema_version": {
-            "hidden comment": "Bump the default in the subclasses."
+            "hidden comment": "Bump the default in the subclasses.",
+            "default": "4"
         },
-        "age": {
-            "title": "Age",
-            "description": "Age of organism at collection of the biosample.",
-            "comment": "Ages over 90 years are generic for privacy purposes.",
-            "type": "string",
-            "pattern": "^((\\d+(\\.[1-9])?(\\-\\d+(\\.[1-9])?)?)|(unknown)|([1-8]?\\d)|(90 or above))$"
+        "lower_bound_age": {
+            "title": "Lower Bound Age",
+            "description": "Lower bound of age of the organism at the time of collection of the sample.",
+            "comment": "For samples from human donors older than 89 years, the lower bound of age has to be 90 and upper bound has to be 90, for de-identification purposes.",
+            "type": "number"
+        },
+        "upper_bound_age": {
+            "title": "Upper Bound Age",
+            "description": "Upper bound of age of the organism at the time of collection of the sample.",
+            "comment": "For samples from human donors older than 89 years, the lower bound of age has to be 90 and upper bound has to be 90, for de-identification purposes.",
+            "type": "number"
```

Comment:
Can we use max value here to limit to 90?  Or likely in the dependencies to limit to 90 for human.

Reply:
I don't think a max would work here as this maximum only applies to ages in age_units years.

Reply:
For example, a biosample can be 100 months old but not 100 years old.

Reply:
Could we add it in a dependency?  Maybe not